packages/framework/attributor/package.json

```diff
@@ -100,8 +101,10 @@
 		"typescript": "~4.5.5"
 	},
 	"typeValidation": {
-		"disabled": true,
-		"version": "2.1.0",
+		"version": "2.0.0-internal.3.2.0",
```

Comment:
@Abe27342, based on your PR description in #13419, this probably should be `@fluid-experimental/attributor` instead of `@fluidframework/attributor`, but published and with no typetesting.  I see everything is still marked as `@alpha`, so experimental scope is more appropriate here.

Reply:
It seems like we're adding attributor capabilities to other DDSes: #13989

That implies it can't be in fluid-experimental, but it raises more questions.

Reply:
I'll file some work to enable the typetests. (btw, do we have a good flow for turning those on the moment we publish? It would be a great thing to document, I couldn't figure it out and there was some dust settling in the area at the time I un-privated it). Publishing is intentional, we're at the point where we want partner teams to start playing around with the APIs but not commit to back-compat in semver so that we're not tied to a particular release schedule in case they have feedback which necessitates a breaking change. Maybe [beta](https://api-extractor.com/pages/tsdoc/tag_beta/) is a better tag, but not sure we have precedent for that in our repo or guidelines that APIs marked beta can be changed without breaking notices.

Reply:
oh, I thought this was on a different attributor PR and only replied to the comment out of context. Enabling them at this point seems reasonable to me. The usefulness of tooling validate any breaking changes are intentional at this point outweighs the annoyance of having to update it in early API iteration. Let me know if you have any other questions that weren't answered above--happy to discuss.